### PR TITLE
HPCC-17053 Change in Dropzone Config

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17053 Change in Dropzone Config
Document the changes in the Dropzone configuration
in the Using the Configmgr doc.
Note: also added logic to automatically generate the attribute table from the XSD file. 

@JamesDeFabia please review
@rpastrana please review

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by JamesDeFabia** [comment]
Looks good from my POV

File: docs/UsingConfigManager/UsingConfigManager.xml [hunk 1]
```diff
@@ -1119,19 +1121,35 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
           <title>DropZone Server List</title>
 
           <para>This tab allows you to add any servers that you wish to
-          configure as drop zones. To add a server to the available drop
-          zones. <orderedlist>
+          configure as a part of this drop zone. To add a server to the
```
Comment (rpastrana) [inline docs/UsingConfigManager/UsingConfigManager.xml]: the last sentence seems incomplete, you might want to terminate it with a ":" and maybe place it in its own line.

File: docs/UsingConfigManager/UsingConfigManager.xml [hunk 2]
```diff
@@ -1119,19 +1121,35 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
...
+              </listitem>
+
               <listitem>
                 <para>Select the <emphasis role="bold">Server List</emphasis>
                 tab.</para>
               </listitem>
 
               <listitem>
-                <para>Double-click on the Server Address field.</para>
+                <para>RIght-click on the Server Address field and choose
```
Comment (rpastrana) [inline docs/UsingConfigManager/UsingConfigManager.xml]: there's no need to capitalize the I in RIght

**Review by rpastrana — CHANGES_REQUESTED** [review]
@g-pan a couple of comments

File: docs/UsingConfigManager/UsingConfigManager.xml (hunk 1 above)
Comment (JamesDeFabia) [inline docs/UsingConfigManager/UsingConfigManager.xml]: I think he meant to end with a colon.
"To add a server to the current drop zone:" 
However, the doc doesn't clarify the difference between adding more servers to one drop zone component and adding additional drop zone components.

**Comment by g-pan** [comment]
Made revisions, new doc can be reviewed at : 
http://10.240.32.243/view/Docs/job/DocDev05/2/CE=ubuntu-14.04-amd64-1/artifact/build-docs/Release/docs/UsingConfigManager-6.3.0-trunk0.pdf
or download the UsingConfigManager file from:
http://10.240.32.243/view/Docs/job/DocDev05/2/CE=ubuntu-14.04-amd64-1/

**Comment by richardkchapman** [comment]
@rpastrana Back to you

**Comment by rpastrana** [comment]
holding off on review per @g-pan

**Comment by rpastrana** [comment]
@g-pan PDF looks reasonable

**Pull request merged**